src/main/java/net/atos/zac/util/MakeYassonSerializable.java

```diff
@@ -13,5 +13,5 @@
  * "T needs to implement an interface or extends an abstract class. A dummy is fine, it is not added to the schema.
  * For java this means that T can not be a record, in kotlin a data class is possible."
  */
-public interface DummyInterface {
+public interface MakeYassonSerializable {
```

Comment:
```suggestion
public interface SerializableByYasson {
```

Reads a bit nicer I think?

Reply:
Applied